deploy/cluster_role.yaml

```diff
@@ -70,7 +58,13 @@ rules:
   - machines
   - machinesets
   verbs:
-  - '*'
+  - create
```

Comment:
what is excluded here from `*`?

Reply:
Basically all the verbs were mentioned and now reduce it to minimal required one.   
```
  - create
  - delete
  - get
  - list
```

Reply:
I think I saw more verbs listed here before. But it looks fine now

Reply:
@bmeng in my last commit i have update this with minimal required verbs. :)